FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -237,6 +246,110 @@ - (void)callListeners:(NSString *)key config:(NSDictionary *)config {
   }
 }
 
+- (void)setCustomSignals:(nullable NSDictionary<NSString *, NSObject *> *)customSignals
+          WithCompletion:(void (^_Nullable)(NSError *_Nullable error))completionHandler {
```

Comment:
i think this should be lowercase `withCompletion`

Reply:
Thanks, I have updated it.